patches/chromium/.patches

```diff
@@ -141,3 +141,5 @@ revert_partial_remove_unused_prehandlemouseevent.patch
 allow_electron_to_depend_on_components_os_crypt_sync.patch
 expose_referrerscriptinfo_hostdefinedoptionsindex.patch
 chore_disable_protocol_handler_dcheck.patch
+fix_check_for_file_existence_before_setting_mtime.patch
+revert_cleanup_remove_feature_windelayspellcheckserviceinit.patch
```

Comment:
I've created an upgrades followup task for the spellcheck patch: https://github.com/electron/electron/issues/48635